client/my-sites/checkout/checkout-thank-you/marketplace/use-themes-thank-you-data.tsx

```diff
@@ -94,6 +101,7 @@ export function useThemesThankYouData(
 	const isAtomicNeeded = hasDotOrgThemes || hasExternallyManagedThemes;
 
 	return [
+		themesList,
```

Comment:
This is a bit confusing for me. We're returning a themeList but the consumers of this hook get the first theme from the list and use it as the selected/installed theme. Could we perform that logic inside this hook instead and instead of returning a list of themes we can return the selected/installed theme instead? That would simplify the code by separating concerns.